src/utils/backend.js

```diff
@@ -116,7 +116,11 @@ export default class Backend {
     if (ordering) queryParams.set('ordering', ordering);
     if (direction) queryParams.set('direction', direction);
     if (search) queryParams.set('search', search);
-    return backendFetch(`tokenCache/wordRegistry?${queryParams.toString()}`);
+    return backendFetch(`tokenCache/wordRegistry?${queryParams.toString()}`)
+      .catch((err) => {
+        console.log(`tokenCache/wordRegistry response with: ${err}`);
+        return [];
+      });
```

Comment:
proposing to move it to where it is called because this code shouldn't be responsible for error handling

Reply:
Why we can't later to wrapp the api methods with the same wrapTry function to catch all errors at one place? Like 
```js
backendFetch = (method) => wrapTry(sdk.api[method])
```

Reply:
@davidyuk currently it used by 4 files https://github.com/aeternity/superhero-ui/search?q=Backend.getWordRegistry and no one of that fetch is works fine, that's the reason why I defined it here. And the return empty array is currently for that like https://github.com/aeternity/superhero-ui/blob/ddaeb71bc9c2fb9886d9eafeef989b57a1f28a13/src/store/index.js#L57

Reply:
> Why we can't later to wrapp the api methods with the same wrapTry function to catch all errors at one place? 

because it is a bad practice

3 or 4 use cases are not relevant, fix only the call in App.vue